src/structured-text/structured_text_language.h

```diff
@@ -0,0 +1,115 @@
+/*******************************************************************\
+
+Module: Structured Text Language Interface
+
+Author: Matthias Weiss, matthias.weiss@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Structured Text Language Interface
+
+#ifndef CPROVER_STRUCTURED_TEXT_STRUCTURED_TEXT_LANGUAGE_H
+#define CPROVER_STRUCTURED_TEXT_STRUCTURED_TEXT_LANGUAGE_H
+
+#include "structured_text_parse_tree.h"
+
+#include <langapi/language.h>
+#include <util/make_unique.h>
+
+/// Implements the language interface for the Structured Text language.
+/// Includes functions for parsing input streams and for converting the
+/// resulting parse tree into a symbol table.
+class structured_text_languaget : public languaget
+{
+public:
+  /// Sets language specific options.
+  void set_language_options(const optionst &options) override;
```

Comment:
No need to include parameter names when the documentation doesn't refer to them.